src/applications/lgy/coe/containers/introduction-content/COEIntroPageBox.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import COEAvailable from './COEStatuses/COEAvailable';
+import COEAutomatic from './COEStatuses/COEAutomatic';
+import COEPending from './COEStatuses/COEPending';
+import COEDenied from './COEStatuses/COEDenied';
+
+const COEIntroPageBox = props => {
+  let content;
+  if (props.coe.status) {
+    switch (props.coe.status) {
+      case 'available':
+        content = <COEAvailable />;
+        break;
+      case 'eligible':
+        content = <COEAutomatic />;
+        break;
+      case 'pending' || 'pending-upload':
+        content = <COEPending status={props.coe.status} />;
+        break;
+      case 'denied':
+        content = <COEDenied />;
+        break;
+      default:
```

Comment:
We should provide a default case, even if it's just a loading indicator of some sort, so we can give the user some sort of visual feedback of what's going on.

Reply:
This already had a default case that returned `<></>` not sure why that didn't show up before